.github/workflows/windows.yml

```diff
@@ -1,4 +1,4 @@
-name: Windows Meson tests
+name: Windows tests
```

Comment:
We're going to have a setup.py job to start with, so remove meson

Reply:
And we are going to only have Meson in the end so we could remove this mention from everywhere.